reconcile/sql_query.py

```diff
@@ -278,14 +332,31 @@ def filesystem_closing_message():
         'echo;',
         'echo Get the sql-query results with:;',
         'echo;',
-        'echo  oc cp ${HOSTNAME}:/tmp/query-result.txt ',
+        'echo  oc cp ${HOSTNAME}:tmp/query-result.txt ',
         '${HOSTNAME}-query-result.txt;',
         'echo;',
         f'echo Sleeping {POD_TTL}s...;',
         f'sleep {POD_TTL};',
     ]
 
 
+def encrypted_closing_message(recipient):
+    return [
+        'cat /gpg/gpg-key | base64 --decode | ',
+        'gpg --homedir /tmp/.gnupg --import;',
+        'gpg --trust-model always --homedir /tmp/.gnupg --armor -r ',
+        recipient,
+        ' --encrypt /tmp/query-result.txt;',
```

Comment:
Will the /tmp/query-result.txt be kept unencrypted in the container? If yes, it shouldn't. The encryption is meant to make sure only the requestor has access to the results... we should remove the plain results file.